docs/release-notes.rst

```diff
@@ -6,6 +6,70 @@
  Release Notes
 ###############
 
+**************
+ Version 0.23
+**************
+
+Version 0.23.0
+==============
+
+**Release Date:** June 05, 2023
+
+**Breaking Changes**
+
+-  Remove HDFS checkpoint storage support, which has been deprecated since 0.21.1 (April, 2023).
+
+-  Kubernetes: When a pod spec is specified in both ``task_container_defaults`` and in the
+   experiment/job configuration, the pod spec is merged according to `strategic merge patch
+   <https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/#use-a-strategic-merge-patch-to-update-a-deployment>`__.
+   The previous behavior was using only the experiment/job configuration if supplied.
+
+-  CLI: `det notebook|tensorboard start` commands no longer block for the whole life cycle of the
+   Notebook or the TensorBoard process. These will also not stream related event logs. Users should
+   use the existing `det notebook|tensorboard|task logs` commands to stream logs from the process.
+
+-  Python SDK: Remove packages ``determined-cli``, ``determined-common``, and ``determined-deploy``,
```

Comment:
```suggestion
-  Python SDK: Remove the packages ``determined-cli``, ``determined-common``, and ``determined-deploy``,
```
I suppose moving "packages" to the front is a reasonable change, at least.

Reply:
Adding "the" sounds weird to me though, but English is not my first language anyway.